frontend/src/lib/stores.js

```diff
@@ -1,5 +1,28 @@
-import words from '../languages/et/words.json'
 import { writable } from 'svelte/store'
+import languages from '../languages/languages.json'
+
+let words = [
+  {
+    source: 'tere',
+    translations: [
+      {
+        word: 'Hi',
+        tag: 'NOUN',
+      },
+    ],
+  },
+]
+
+async function loadWords() {
+  return (await import(`../languages/${_currentLanguage.code}/words.json`)).default
```

Comment:
Make this function check if the currently loaded words are already for the correct language, if they are, then return them. If they are not, then import them and then return them.